HousingRepairsOnlineApi/Gateways/NotifyGateway.cs

```diff
@@ -14,29 +13,15 @@ public NotifyGateway(INotificationClient client)
             this.client = client;
         }
 
-        public async Task<SendSmsConfirmationResponse> SendSms(string number, string templateId,
+        public async Task SendSms(string number, string templateId,
```

Comment:
> This async method lacks 'await' operators and will run synchronously

Please address this warning by either not having the method be asynchronous or making the method code run asynchronously.